src/controllers/crawling.controller.js

```diff
@@ -0,0 +1,106 @@
+const { v4: uuidv4 } = require("uuid");
+const { crawl } = require("../crawler/crawl");
+
+let clients = [];
+let linksQueue = [];
+let shouldStopCrawling;
+let crawlURL;
+
+exports.streamCrawling = async function (req, res) {
+  const headers = {
+    "Content-Type": "text/event-stream",
+    Connection: "keep-alive",
```

Comment:
혹시 Connection도 string 형태로 바꿔도 무방할까요~?
```suggestion
    "Connection": "keep-alive",
```

Reply:
prettier에서 property에 특수문자가 없는 경우는 quotes를 생략하는 것 같습니다!